dev-docs/source/Testing/IndirectInelastic/IndirectInelasticAcceptanceTests.rst

```diff
@@ -0,0 +1,202 @@
+.. _indirect_inelastic_testing:
+
+Indirect Inelastic Testing
+==========================
+
+.. contents::
+   :local:
+
+Data reduction
+--------------
+
+*Preparation*
+
+-  Instrument ``IRIS``
+-  You need access to the ISIS data archive
+
+**Time required 2-5 minutes**
+
+--------------
+
+1.  Open ``Interfaces`` > ``Indirect`` > ``Data reduction``
```

Comment:
I believe it is better to use `#.` for numbered lists as this enables easy addition of new items/shuffling:
```
#. Open ``Interfaces`` > ``Indirect`` > ``Data reduction``
#. Make sure ``Instrument`` is set to ``IRIS``
[...]
```